src/pages/individual/fakeAssets.js

```diff
@@ -1,57 +1,69 @@
 const fakeAssets = [
   {
-    id: 1,
+    number: 1,
```

Comment:
is this file still used? if not lets delete it

Reply:
Done. Sad to see all of my carefully-curated fake animal pelts go.